jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/ImmutableHttpFields.java

```diff
@@ -154,6 +155,12 @@ public HttpField next()
         };
     }
 
+    @Override
+    public ListIterator<HttpField> listIterator(int index)
+    {
+        return new Listerator(index);
```

Comment:
Is this class really called `Listerator`? Seems a far cousin of a dinosaur 🤣

Reply:
I like the name.